tests/PySys/software_management_end_to_end/sm_mapper_fail_and_reconnect/run.py

```diff
@@ -32,11 +32,10 @@ class SmMapperC8yReceiveLastMessageOnRestart(BaseTest):
     rm = "/usr/bin/rm"
     def setup(self):
 
-        self.startProcess(
-            command=self.sudo,
-            arguments=[self.tedge, "connect", "c8y"],
-            stdouterr="connect_c8y",
-        )
+        # Delete and now we wait, so there is a time depencency
```

Comment:
I didn't really understand that comment, on how it's related to what's happening in `smcleanup` which is just a `tedge disconnect`.

Reply:
Thanks, for the hint. I also do not understand that line. It slipped through accidentally.
However, I'll keep the next line, that really was a test issue that caused others to fail.

More info is here:
https://github.com/thin-edge/thin-edge.io/issues/752